ngi_pipeline/engines/piper_ngi/parsers.py

```diff
@@ -155,3 +155,20 @@ def parse_genotype_concordance(genotype_concordance_file):
                                        gt_entry.overall_genotype_concordance))
             continue
     return samples_gtc_dict
+
+def parse_deduplication_percentage(deduplication_file):
+
+    duplication_percentage=0
+    with open(deduplication_file, 'r') as f:
+        for line in iter(f.readline, ''):
+            if "## METRICS CLASS" in line and "picard.sam.DuplicationMetrics" in line:
+                try:
+                    headers=f.readline()
+                    values=f.readline()
+                    duplication_rate=values.split()[7]
```

Comment:
Also in this case I would prefer to be less stringent/hardcoded. Loop over the headers and find out which one is the duplication we are interested (i.e., `PERCENT_DUPLICATION`).